components/tensorflow-notebook-image/Dockerfile

```diff
@@ -16,7 +16,7 @@ ENV DEBIAN_FRONTEND noninteractive
 ENV NB_USER jovyan
 ENV NB_UID 1000
 ENV HOME /home/$NB_USER
-ENV CONDA_DIR=$HOME/.conda
+ENV CONDA_DIR=/opt/conda
```

Comment:
Can you add a comment explaining why we use /opt/conda.

Reply:
Done.